_meta/beat.yml

```diff
@@ -5,7 +5,7 @@
 gpfsbeat:
   # Defines how often an event is sent to the output
   period: 60s
-  filesystem: ["/"]
+  device: ["/"]
```

Comment:
this makes no sense; but i do not think there's a reasonable default.
you can do `mmrepquota -Y all` but, the output has lines like
```
*** Report for USR GRP FILESET quotas on scratchdelcatty
mmrepquota::HEADER:version:reserved:reserved:filesystemName:quotaType:id:name:blockUsage:blockQuota:blockLimit:blockInDoubt:blockGrace:filesUsage:filesQuota:filesLimit:filesInDoubt:filesGrace:remarks:quota:defQuota:fid:filesetname:
mmrepquota::0:1:::scratchdelcatty:USR:0:root:42301440:0:0:655360:none:2478:0:0:3:none:i:on:off:0:root:
...
*** Report for USR GRP FILESET quotas on scratchphanpy
mmrepquota::HEADER:version:reserved:reserved:filesystemName:quotaType:id:name:blockUsage:blockQuota:blockLimit:blockInDoubt:blockGrace:filesUsage:filesQuota:filesLimit:filesInDoubt:filesGrace:remarks:quota:defQuota:fid:filesetname:
mmrepquota::0:1:::scratchphanpy:USR:0:root:536092928:0:0:2494089344:none:16:0:0:80:none:i:on:off:0:root:
m
...
```
so make sure your parser doesn't explode 😄